src/SQLiteAsync.cs

```diff
@@ -77,7 +79,9 @@ public SQLiteAsyncConnection (string databasePath, bool storeDateTimeAsTicks = t
 		public SQLiteAsyncConnection (string databasePath, SQLiteOpenFlags openFlags, bool storeDateTimeAsTicks = true)
 		{
 			_openFlags = openFlags;
+			isFullMutex = _openFlags.HasFlag (SQLiteOpenFlags.FullMutex);
 			_connectionString = new SQLiteConnectionString (databasePath, storeDateTimeAsTicks);
+			_connection = new SQLiteConnectionWithLock (_connectionString, openFlags) { SkipLock = isFullMutex };
```

Comment:
This `_connection` is only used if we are full mutex, right? So let's only create it if that flag is set?

Also, perhaps it should be renamed `_fullMutexReadConnection` since it's never used for writing and I don't want to confuse it with the pool.